sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/api/Azure.ResourceManager.RecoveryServicesBackup.netstandard2.0.cs

```diff
@@ -2588,6 +2594,18 @@ public partial class TargetAfsRestoreInfo
         public string Name { get { throw null; } set { } }
         public Azure.Core.ResourceIdentifier TargetResourceId { get { throw null; } set { } }
     }
+    public enum TargetDiskNetworkAccessOption
+    {
+        SameAsOnSourceDisks = 0,
+        EnablePrivateAccessForAllDisks = 1,
+        EnablePublicAccessForAllDisks = 2,
+    }
+    public partial class TargetDiskNetworkAccessSettings
```

Comment:
Could we add a prefix to these two types to emphasize its ownership?
Other RPs also have similar things and we require all the classes to have unique names across all SDKs

Reply:
updated as per suggestion